samples/net/gsm_modem/src/main.c

```diff
@@ -6,15 +6,59 @@
 
```

Comment:
The commit "samples: net: gsm: Add suspend/resume shell commands" could be removed from the PR, I can submit it separately after this PR is merged.